### DISCUSSION TITLE
Grafana Alerting - Include image of visualized query with the alert notification

### DISCUSSION THREAD

**Discussion by borghol** [grafana/grafana#41015] [Alerting]
**What happened**:
- In legacy alerting (<8), a screenshot of the panel was included in email (No external image storage). However, after opting-in to Grafana 8 alerts, screenshots of the query no longer appears in the email.
- Legacy alerting showed the offending value as well (AKA Query X generated a value of Y)
![image](https://user-images.githubusercontent.com/8896043/138812652-dc770122-5d5c-4a56-bfd2-1955dd068cad.png)

**What you expected to happen**:
A screenshot of the query through the image renderer as well as the value should be in the email

**How to reproduce it (as minimally and precisely as possible)**:
- Create a new alert rule through the Unified Alerting section (with email & image renderer set up)
- Wait for the alert to trigger
- Check your email :D

**Anything else we need to know?**:
Checking the metrics of the image renderer, I found that there is no HTTP request received

**Environment**: 
- Grafana version: 8.2.2
- Data source type & version: Prometheus / CloudWatch
- OS Grafana is installed on: Docker (Image Renderer on Docker as well)
- User OS & Browser: -
- Grafana plugins: None
- Others: AWS hosted

**Comment by pkolyvas** [comment]
Hey there @borghol - thanks for the feedback. You're correct in noting that our new unified alerting system doesn't include images/renders of the queries in v8. Your feature request is valuable feedback that will help us prioritize what features we bring over from our legacy alerting system moving forward. 

To give you some context on why this particular feature didn't carry over - Grafana 8 alerts are no longer directly tied to panel queries. Alerts can, in some cases, be linked to panels - but they are separate queries. We're hopeful that this brings some added flexibility to the way teams create/manage and handle alerts but it does mean there are workflows changes that may not be idea for all use cases or expectations. 

For the "value in the notification" - you should be able to customize a template, but to your point it could be valuable to have the query values as part of the default template. Would you be willing to create a separate thread for that?

**Reply by borghol** [reply]
Done. #41019

**Reply by pkolyvas** [reply]
Thanks!

**Comment by gotjosh** [comment]
Locking in favour of https://github.com/grafana/grafana/discussions/38030
